src/components/TabsPage.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import classNames from 'classnames';
+import {
+  Link,
+  useParams,
+} from 'react-router-dom';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tabs: Tab[] | undefined;
+};
+
+export const TabsPage: React.FC<Props> = ({ tabs }) => {
+  const { tabId } = useParams();
+  const isSelected = (tab: Tab) => tab.id === tabId;
+  const tabContent = () => {
+    return tabs?.find(tab => tab.id === tabId);
+  };
```

Comment:
you can use it as constant not function
```suggestion
  const tabContent = tabs?.find(tab => tab.id === tabId);
```